internal/database/security_event_logs.go

```diff
@@ -108,21 +113,44 @@ func (s *securityEventLogsStore) InsertList(ctx context.Context, events []*Secur
 
 	if _, err := s.Handle().ExecContext(ctx, query.Query(sqlf.PostgresBindVar), query.Args()...); err != nil {
 		return errors.Wrap(err, "INSERT")
+	} else if auditLogEnabled() {
```

Comment:
```suggestion
	}
	
	if auditLogEnabled() {
```

There's an early return in the other branch, so I'd use this, since it's easy to miss conditions in `else if`.

Reply:
Also: that function doesn't seem to exist?

Reply:
Ah, I just pushed Robert's suggestion and didn't fix the usage. Applying the suggestion otherwise, agreed with the simplification.